webpack.config.js

```diff
@@ -52,9 +52,9 @@ module.exports = (env, argv) => {
             library: dashLibraryName,
             libraryTarget: 'window',
         },
-        devtool,
         externals,
         module: {
+            noParse: /node_modules\/plotly.js/,
```

Comment:
We know the output of Plotly.js is (1) valid, (2) self-contained (doesn't it to resolve outward facing define/import/require), so don't parse it, just take the file as-is.